web-client/src/presenter/actions/updateMessageModalAttachmentsAction.ts

```diff
@@ -25,31 +33,31 @@ export const updateMessageModalAttachmentsAction = ({
 
     const isSubjectEmpty = !subject || subject.trim() === '';
 
+  if (
+    isSubjectEmpty &&
+    (attachments?.length || 0) + (draftAttachments?.length || 0) === 0
```

Comment:
for readability of this, I'd recommend pulling out some of this to variables then pass into the if statement

```
const attachmentCount = attachments?.length || 0;
const draftAttachmentCount = draftAttachments?.length || 0;
const hasNoAttachments = (attachmentCount + draftAttachmentCount) === 0;
```